brooklyn-server/software/winrm/src/main/java/org/apache/brooklyn/util/core/internal/winrm/WinRmTool.java

```diff
@@ -66,8 +66,38 @@
             "Size of file chunks (in bytes) to be used when copying a file to the remote server", 
             1024);
 
+    /**
+     * Executes a Native Windows command.
+     * It is creating a new Shell on the destination host each time it is being called.
+     * @param command
+     * @since 0.2
+     */
+    WinRmToolResponse executeCommand(String command);
+
+    /**
+     * Executes a Power Shell command.
+     * It is creating a new Shell on the destination host each time it is being called.
+     * @param psCommand
+     * @since 0.2
+     */
+    WinRmToolResponse executePsCommand(String psCommand);
+
+    /**
+     * Execute a list of Windows Native commands as one command.
+     * The method translates the list of commands to a single String command with a <code>"\r\n"</code> delimiter and a terminating one.
+     * @param commands
+     * @deprecated since 0.2; Use the {@link #executeCommand(String)} instead and transform your commands list explicitly
+     */
+    @Deprecated
     WinRmToolResponse executeScript(List<String> commands);
```

Comment:
@bostko see @neykov 's comments in https://github.com/cloudsoft/winrm4j/pull/2, saying "we should keep the `(List<String>)`...".